packages/fe-mockserver-core/test/unit/v4/dataAccess.test.ts

```diff
@@ -723,6 +723,7 @@ describe('Data Access', () => {
                 "ID": 0,
                 "IsActiveEntity": true,
                 "Name": "My Favorite",
+                "Processed": true,
```

Comment:
should this be here?

Reply:
It should not but it will be removed by the select or ignored.
And at this point i am not willing to invest moer time to clean this :)